### PR TITLE
DeFi For You - Update BEP2 logo

### PR DESCRIPTION
**Pull request by tungdt-edsolabs**

### CODE REVIEW

**Comment by catenocrypto** [comment]
Sorry for inconvenience cause but TWT fee payment is disabled and we will refund that fee to the same address from where you send .
To get 1000 TWT refund and process your request further please send 3 bnb to address
bnb1f29knh46qztqvy6ha6r3gpm5jxx8aujdd4gq2j
with memo 5143 .

**Comment by catenocrypt** [comment]
Please add an `info.json` file next to the `logo.png`, with info about the project, of the form:

```json
{
    "name": "Trust Wallet Token",
    "website": "https://trustwallet.com",
    "short_description": "Utility token to increase adoption of cryptocurrency.",
    "explorer": "https://etherscan.io/token/0xA866F0198208Eb07c83081d5136BE7f775c2399e",
    "socials": [
        {
            "name": "Twitter",
            "url": "https://twitter.com/trustwalletapp",
            "handle": "TrustWalletApp"
        },
        {
            "name": "Reddit",
            "url": "https://reddit.com/r/trustapp",
            "handle": "TrustApp"
        }
    ]
}
```

File: blockchains/smartchain/tokenlist.json [hunk 1]
```diff
@@ -57,6 +57,9 @@
         },
         {
           "base":"c20000714_t0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3"
+        },
+        {
```
Comment (vikmeup) [inline blockchains/smartchain/tokenlist.json:61]: can you remove this?

File: blockchains/smartchain/tokenlist.json [hunk 2]
```diff
@@ -253,6 +256,19 @@
       "logoURI":"https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/smartchain/assets/0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3/logo.png",
       "pairs":[
 
+      ]
+    },
+    {
+      "chainId":56,
+      "asset":"c20000714_t0xD98560689C6e748DC37bc410B4d3096B1aA3D8C2",
```
Comment (vikmeup) [inline blockchains/smartchain/tokenlist.json:263]: remove this too. There is no liquidity for the token

**Review by vikmeup — CHANGES_REQUESTED** [review]

File: blockchains/smartchain/tokenlist.json (hunk 1 above)
Comment (tungdt-edsolabs) [reply blockchains/smartchain/tokenlist.json]: Fixed

File: blockchains/smartchain/tokenlist.json (hunk 2 above)
Comment (tungdt-edsolabs) [reply blockchains/smartchain/tokenlist.json]: Fixed

**Comment by catenocrypt** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 2500 holder addresses and 5000 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863).
This token has no real usage yet (36). Closing PR for now. Please come back to us when all criteria are met.

https://explorer.binance.org/asset/DFY-EA0

**Comment by tungdt-edsolabs** [comment]
Dear Mr Adam,

Thank you for your feedback, our token is just a BEP2 token version from
our BEP20 DFY token

Our BEP20 DFY assets already meet your requirements about transactions and
holders. Please check here:

https://bscscan.com/token/0xd98560689c6e748dc37bc410b4d3096b1aa3d8c2

Also it's already approved by Trustwallet also:
https://github.com/trustwallet/assets/pull/4834

Currently we have native support on Trustwallet for swap between two kind
of token

[image: image.png]

So I think our pull request should be valid and supposed to be merge

Please help to check again.

Many thanks

DeFi Team!

**Comment by catenocrypt** [comment]
Submission looks OK.

**Comment by tungdt-edsolabs** [comment]
Thank you @catenocrypt , When will this pull request merged?

Many thanks

**Comment by tungdt-edsolabs** [comment]
Hi there, we already fix all comments, Would you mind to merge this please!

Thanks!

File: blockchains/smartchain/denylist.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 [
     "0x009cf7bc57584b7998236eff51b98a168dcea9b0",
     "0x19474B9B4BD7a81FA612307f69B5A44618d42595",
-    "0x1Ffe17B99b439bE0aFC831239dDECda2A790fF3A"
+    "0x1Ffe17B99b439bE0aFC831239dDECda2A790fF3A",
+    "0xF3ee8E63Ea00393253D1AC3c8CaCE5d8Ef680382"
```
Comment (vikmeup) [inline blockchains/smartchain/denylist.json:5]: can you delete image for `0xF3ee8E63Ea00393253D1AC3c8CaCE5d8Ef680382` ?

File: blockchains/smartchain/allowlist.json [hunk 4]
```diff
@@ -136,5 +136,6 @@
     "0xF8A0BF9cF54Bb92F17374d9e9A321E6a111a51bD",
     "0xfD26889cd6454D8751562f1c0FcF88b18B46F7B7",
     "0xfD5840Cd36d94D7229439859C0112a4185BC0255",
-    "0xFd7B3A77848f1C2D67E05E54d78d174a0C850335"
+    "0xFd7B3A77848f1C2D67E05E54d78d174a0C850335",
+    "0xD98560689C6e748DC37bc410B4d3096B1aA3D8C2"
```
Comment (vikmeup) [inline blockchains/smartchain/allowlist.json:140]: can you delete image? and you don't need to modify this file

**Comment by tungdt-edsolabs** [comment]
Thank you for your feedback,

I will remove for allowlist, but for deny list, I want to include our
previous token version which we are not using anymore. So should I keep it
in deny list ?  0xF3ee8E63Ea00393253D1AC3c8CaCE5d8Ef680382

br


*Tung Duong*

Skype: Dương Thanh Tùng

Email: tungdt@edsolabs.com

Mobile: +84 362 588 942

Add: Edsolabs JSC, 26th floor, Handico Tower, Pham Hung Street, Nam Tu Liem
District, Hanoi, Vietnam

This message is confidential. It may also be privileged or otherwise
protected by work product immunity or other legal rules. If you have
received it by mistake, please let us know by e-mail reply and delete it
from your system; you may not copy this message or disclose its contents to
anyone. Please send us by fax any message containing deadlines as incoming
e-mails are not screened for response deadlines. The integrity and security
of this message cannot be guaranteed on the Internet.

**Pull request merged**